### DISCUSSION TITLE
Bandolier has become completely useless.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14170] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I’ve never used a bandolier before because the bot doesn’t remove the bandoleer to carry out an order “prepare for an expedition.” He simply ignores the order "prepare for expedition" then.
Now I won't use a bandoleer either.
1. The bandolier stopped adding +15 to weapon skill.
2. The bandolier cannot be worn simultaneously with an assault rifle, heavy machine gun, or arc emitter.
What's the point of having a bandoleer if you can't carry it with your weapon?
Please, lets make a slot for weapons in the bandoleer so that an assault rifle or machine gun can be placed in the bandoleer. Then it will make sense.
In fact, the bandoleer is a "Tactical Load-Bearing Vest".
Tell us why you can’t wear a tool belt or backpack over it?
Tell us why you can't sling an assault rifle on your back when you're wearing a "Tactical Load-Bearing Vest"?
Tactical Load-Bearing Vest + assault rifle on your back is the basis of combat equipment, is'nt it?
3. The bandolier occupies a regular slot in the closet. It would be logical if it fit in the container slot.
For example, you need to take a bandoleer instead of a belt, but all container slots (in closet) are occupied. To take the bandoleer you will have to throw the belt on the ground.

### Reproduction steps

1. wear bandolier 
2. You can hold assault rifle/HMG only in your hands now.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
> The bandolier stopped adding +15 to weapon skill.

The bandolier was more used as a crafting quality booster before than anything else. There's a lot of sources for extra weapon skill, and another +15 just made skill gains largely irrelevant

> Tell us why you can’t wear a tool belt or backpack over it?

Technically you could also wear a tool belt and backpack both in real life. This is not in real life, and there's good balance & gameplay reasons for that, imo.

> What's the point of having a bandoleer if you can't carry it with your weapon?

Those heavy weapons are an exception, but you can still hold those in your hand while you load yourself with ammo in the Bandolier.

Anyways, I don't see anything here worth addressing, closing.

**Reply by pavel4win** [reply]
It's up to you how you feel about this letter, but I wouldn't recommend closing the discussion right away. First look at the public's reaction

**Reply by Regalis11** [reply]
Actually, we've gotten tons of positive feedback about the change on Steam comments, Discord and Reddit, so it seems at least a sizable portion of the player base consider the changes to be an improvement. I get your points too, but I would argue it now serves a clear purpose as an alternative for toolbelts and such, but for holding large amounts of ammo. It's true that it also means you can't wear some weapons on your back at the same time, but I think it's important to note that those are some of the most powerful handheld weapons in the game, and I feel it's good that they come with serious drawbacks.

It's also worth noting that you posted this in "bug reports", and it's not exactly a bug report but more of an opinion on the recent changes, so in my opinion there's no need to keep the ticket open. Discussion about the balance, and this bandolier change, is more than welcome elsewhere though, and we'll definitely be keeping an eye out for the public's opinion and readjust it if needed!

**Reply by pavel4win** [reply]
ok, I will move the discussion in Ideas and Suggestions.
I created this discussion in bugs, to draw your attention to the fact that the bandolier should be placed in the closet in the container compartment like a tool belt.
The first and second points are IMHO.
But the third one obviously needs to be fixed.
Is it an imbalance if you can put 2 times more ammunition in a cabinet than before by filling the cabinet with bandoliers?
